Libraries/libbom/Tools/lsbom.cpp

```diff
@@ -455,7 +431,12 @@ main(int argc, char **argv)
             printf("%s\n", path.c_str());
         } else {
             // TODO: Respect options about what to print.
-            printf("%s\n", path.c_str());
+            std::stringstream octConvertingStream;
+            octConvertingStream << std::oct << ntohs(path_info_2_value->mode);
+            std::string mode = octConvertingStream.str();
```

Comment:
I would call this just `stream`, pretty clear it's being used for conversion from the `std::oct` in there.